### PR TITLE
First commit to nodeProxy

### PR DESCRIPTION
**Pull request by bernokl**
Investigating nodeProxy as potential conformance endpoint tests

### CODE REVIEW

File: tickets/k8s/proxyNode_v1.org [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+         connectCoreV1PatchNodeProxy,
+         connectCoreV1OptionsNodeProxyWithPath,
+         connectCoreV1OptionsNodeProxy,
+         connectCoreV1HeadNodeProxyWithPath,
+         connectCoreV1HeadNodeProxy,
+         connectCoreV1GetNodeProxy,
+         connectCoreV1DeleteNodeProxyWithPath,
+         connectCoreV1DeleteNodeProxy
+         )
+        ;
```
Comment (bernokl) [inline tickets/k8s/proxyNode_v1.org:270]: @zachmandeville Can you help me find the correct for this query

Comment (zachmandeville) [reply tickets/k8s/proxyNode_v1.org:270]: The list of values in your where claus `operation id in ()` are being read as expected columns in the audit_event table.  If you surround each one with `'` they'll be read as strings, as you intended.

I would recommend running the query again without this clause, to verify that you are getting useragents returned.

**Comment by hh** [comment]
@spiffxp said that if we explore proxy and write a good parameterised test for all proxy, we would cover A-LOT of endpoints.

I'll take this on myself.

/assign @hh

**Comment by Riaankl** [comment]
addressed by Stephen in Q3

**Pull request closed without merging**